packages/ai/src/squiggle/squiggleAutomaticChanges.ts

```diff
@@ -0,0 +1,14 @@
+// replace |> with ->
+function replacePipeWithArrow(code: string): string {
+  return code.replace(/\|>/g, "->");
```

Comment:
Huh, does this still happen often?

I can't find any mentions of the old `|>` anywhere in the docs, did Claude read pre-0.8.0 Squiggle code somewhere on the internet?